lib/sqlalchemy/dialects/postgresql/array.py

```diff
@@ -21,13 +26,25 @@
 from ... import util
 from ...sql import expression
 from ...sql import operators
-from ...sql._typing import _TypeEngineArgument
 
+if TYPE_CHECKING:
+    from ...engine.interfaces import Dialect
+    from ...sql._typing import _TypeEngineArgument
+    from ...sql.elements import ColumnElement
+    from ...sql.elements import Grouping
+    from ...sql.expression import BindParameter
+    from ...sql.operators import OperatorType
+    from ...sql.selectable import _SelectIterable
+    from ...sql.type_api import _BindProcessorType
+    from ...sql.type_api import _LiteralProcessorType
+    from ...sql.type_api import _ResultProcessorType
+    from ...sql.type_api import TypeEngine
 
-_T = TypeVar("_T", bound=Any)
 
+_T = TypeVar("_T", bound=Any_)
 
-def Any(other, arrexpr, operator=operators.eq):
+
+def Any(other, arrexpr, operator=operators.eq):  # type: ignore[no-untyped-def]
```

Comment:
may as well type this instead of ignoring it. same below

Reply:
I would say:
~~~python
def Any(other: Any, arrexpr, operator: OperatorType = operators.eq) -> ColumnElement[bool]: ...
~~~
but it's not clear to me what `arrexpr` should be?

Reply:
I went on with `arrexpr: ColumnElement[Any]` .

Reply:
sorry, missed this. You can look at these: https://github.com/sqlalchemy/sqlalchemy/blob/main/lib/sqlalchemy/sql/_elements_constructors.py

it seems that it uses `_ColumnExpressionArgument[_T]`

Reply:
Thank you for the hint; fixed.